### PR TITLE
404 error page

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: 404-error.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php include 'includes/header.php'; ?>
+    <div class="hero-image full-width bottom-spacer">
```
Comment (ElXyfer) [inline 404-error.php]: Remove hero image.

File: blog.php [hunk 2]
```diff
@@ -21,4 +21,5 @@
     </div> <!-- End of twitter-feed -->
 </div> <!-- End of blog-sidebar-component -->
 
-<?php include 'includes/footer.php';?>
\ No newline at end of file
+<?php include 'includes/footer.php';?>
```
Comment (ElXyfer) [inline blog.php]: Remove footer

File: theme/sass/components/_hero-image.scss [hunk 3]
```diff
@@ -11,6 +11,10 @@
     background-image: url("/theme/img/child-development-childrens-charity-hero.jpeg");
   }
 
+  .page-404-error & {
```
Comment (ElXyfer) [inline theme/sass/components/_hero-image.scss]: You can remove this.

File: 404-error.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    </div> <!-- End hero image -->
+    <div class="error-page-container bottom-spacer">
+        <h1>Ooops!</h1>
+        <h2>The requested URL was not found on this server.</h2>
+        <p>Error code: 404</p>
+        <ul class="error-page-links">
+            <li>Here are some useful links:
```
Comment (ElXyfer) [inline 404-error.php]: Add homepage link.

File: 404-error.php (hunk 1 above)
Comment (katlazarova) [reply 404-error.php]: done

File: blog.php (hunk 2 above)
Comment (katlazarova) [reply blog.php]: done

File: theme/sass/components/_hero-image.scss (hunk 3 above)
Comment (katlazarova) [reply theme/sass/components/_hero-image.scss]: done

File: 404-error.php (hunk 4 above)
Comment (katlazarova) [reply 404-error.php]: done

**Pull request merged**